osquery/core/init.cpp

```diff
@@ -603,7 +603,7 @@ void Initializer::start() const {
     if (tool_ == ToolType::DAEMON) {
       LOG(WARNING) << message;
     } else {
-      LOG(INFO) << message;
```

Comment:
I was thinking that it would be nice to have the option not to log this (i.e. when verbose logging is turned off), instead of the `LOG(INFO)` default behavior of always logging to `/tmp/<program name>...`

In particular, if the volume of logging throughout osquery continues to increase in the future, it would be useful to have as much "non-essential" stuff migrated to verbose logging such as to not clog up the log files and prevent any (marginal) performance issues.

Of course, a single logging statement probably doesn't have much effect in the long run, and I can revert this change if necessary.